test/shoes_spec_helper.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+# This is intended to be required from inside a Shoes-Spec test, which allows writing test
+# helper code for them.
+
+module TextDrawableHelper
+  def trim_html_ids(s)
```

Comment:
So it's better to have class="id_7" than id="7"? That's one of the big changes I see in the fixtures. Just want to ensure my understanding.